README.md

```diff
@@ -101,11 +101,16 @@ Available files, and their formats:
      * `discarded` if it dropped the whole pair
      * Plus a `settings` file with info on how cleaning went
 
-* `ribocounts/`: Intermediate, processed output of `RiboDetector`
+* `ribocounts/`: Intermediate, counts of `RiboDetector` output
```

Comment:
Is it worth keeping this?  Since we can easily get the count by just checking the number of lines in a decompressed `ribopass-reads/` file.

Reply:
I'm fine removing them. Do you have a take on whether we should save the read IDs for non-rRNA reads or for rRNA reads? 

Currently I'm saving read-IDs of non-rRNA sequences since this is the default output of RiboDetector. 

When we want to display % rRNA reads on the dashboard I will need to combine this with number of total reads.

Reply:
No preference on which read ids to keep; it's easy enough to get the full list of read ids and compute the complement.

Combining with the total number of reads is also not hard.